website/docs/advanced-concepts/sharing-data-across-pages.md

```diff
@@ -6,41 +6,83 @@ description: >-
 ---
 
 # Share Data Across Pages
-You can share data between pages by utilizing query parameters or Appsmith's store value, which is a global state object. This page explains how to share data between different pages in your Appsmith applications.
 
-## Use query parameters
-Use the query parameter to share data between pages across apps. To pass data between the source and the target page using query parameters, use the [navigateTo()](/reference/appsmith-framework/widget-actions/navigate-to) function.
+This guide shows how to share data across different pages in Appsmith using the `navigateTo()` or the `storeValue()` function.
 
-Example:
+## Using navigateTo() Function
```

Comment:
Shouldn't this just be Examples -- why we need using navigateTo() function as heading?